src/procontroller_snowbots/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>procontroller_snowbots</name>
+  <version>0.0.0</version>
+  <description>The procontroller_snowbots package</description>
+  <maintainer email="kevinlinxc@gmail.com">kevinlinxc</maintainer>
+  <license>TODO</license>
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>geometry_msgs</build_depend>
+  <build_depend>roscpp</build_depend>
+  <build_depend>libevdev-dev</build_depend>
```

Comment:
```suggestion
  <build_depend>libevdev-dev</build_depend>
  <build_depend>sb_utils</build_depend>
```
See if adding `sb_utils` as a build dependency will ensure it is compiled before `procontroller_snowbots`.